lib/gitlab/client/branches.rb

```diff
@@ -82,7 +88,6 @@ def create_branch(project, branch, ref)
     #
     # @param  [Integer, String] project The ID or name of a project.
     # @param  [String] branch The name of the branch to delete
-    # @return [Gitlab::ObjectifiedHash]
```

Comment:
Sure, I can if you want - but I think it always returns false... maybe there are times it would return true, but I don't know what they would be - do you?  It seems more confusing than helpful to document it without being able to give any description of what it represents. (All ruby functions return something, of course, but is it worth documenting if the return value is meaningless?)  I suppose I could document it as "meaningless return value" or something?